test/Test/HTTPure/HeadersSpec.purs

```diff
@@ -2,11 +2,13 @@ module Test.HTTPure.HeadersSpec where
 
 import Prelude
 
+import Data.Array.NonEmpty (fromArray, singleton)
 import Data.Maybe (Maybe(Nothing, Just))
 import Data.Tuple (Tuple(Tuple))
 import Effect.Class (liftEffect)
-import HTTPure.Headers (empty, header, headers, read, write)
 import HTTPure.Lookup ((!!))
+import HTTPure.RequestHeaders (empty, read)
```

Comment:
It's not clear to me if the tests below that test `empty` are meant to be testing `HTTPure.RequestHeaders.empty` or `HTTPure.ResponseHeaders.empty`. Or perhaps they need to be duplicated so we can test both?